src/plugins/vis_type_timeseries/public/application/components/vis_editor.js

```diff
@@ -36,6 +36,12 @@ import { Storage } from '../../../../../plugins/kibana_utils/public';
 const VIS_STATE_DEBOUNCE_DELAY = 200;
 const APP_NAME = 'VisEditor';
 
+const debouncedFetchFields = debounce(
```

Comment:
- Please move it into `VisEditor` class as a field of that class. In case on 2 TSVB instances on the Dashboard this code can be a reason of regression.
- Is it possible to use Abort Controller. I see it like:  
![image](https://user-images.githubusercontent.com/20072247/100448473-6703f280-30c3-11eb-99ef-69a522ea8d8c.png)
and
![image](https://user-images.githubusercontent.com/20072247/100448533-7c791c80-30c3-11eb-8ac9-cab67066cd99.png)